scripts/dev/mint_erc20/main.go

```diff
@@ -0,0 +1,766 @@
+// Package main provides a CLI tool for USDC operations: deposit, balance check, and transfer
+package main
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"flag"
+	"fmt"
+	"github.com/vultisig/vultisigner/common"
+	"log"
+	"math/big"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	gcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+// Constants
+const (
+	// Default USDC contract address on Ethereum mainnet
+
+	derivePath   = "m/44'/60'/0'/0/0"
+	hexChainCode = "1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef"
+
+	// USDC decimals (USDC uses 6 decimals unlike most ERC20 tokens that use 18)
+	WETHDecimals = 18
+
+	// Gas limits
+	DefaultGasLimit = 1000000
+	GasLimitBuffer  = 50000
+
+	DeadlineDuration = 15 * time.Minute
+)
+
+// Contract ABIs
+const (
+	// WETH ABI with deposit and transfer functions
+	WETHABI = `[
+		{
+			"constant": false,
+			"inputs": [],
+			"name": "deposit",
+			"outputs": [],
+			"payable": true,
+			"stateMutability": "payable",
+			"type": "function"
+		},
+		{
+			"constant": false,
+			"inputs": [
+				{
+					"name": "dst",
+					"type": "address"
+				},
+				{
+					"name": "wad",
+					"type": "uint256"
+				}
+			],
+			"name": "transfer",
+			"outputs": [
+				{
+					"name": "",
+					"type": "bool"
+				}
+			],
+			"payable": false,
+			"stateMutability": "nonpayable",
+			"type": "function"
+		},
+		{
+			"constant": true,
+			"inputs": [
+				{
+					"name": "",
+					"type": "address"
+				}
+			],
+			"name": "balanceOf",
+			"outputs": [
+				{
+					"name": "",
+					"type": "uint256"
+				}
+			],
+			"payable": false,
+			"stateMutability": "view",
+			"type": "function"
+		}
+	]`
+)
+
+var vaultName string
+var stateDir string
+var WETHAddr = gcommon.HexToAddress("0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2")
```

Comment:
duplicate of `WETHAddress = gcommon.HexToAddress("0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2")`